### PR TITLE
Client fours bugs

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: frontend/src/app/components/janus/janus.service.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export class JanusService {
       if (!Janus.isWebrtcSupported()) {
         this.toastr.error('Sorry, we need your browser to support WebRTC.');
         deffered.reject();
+        return;
```
Comment (kkavod) [inline frontend/src/app/components/janus/janus.service.ts]: Should be
return deferred. promise

File: frontend/src/app/components/janus/janusVideoRoom.service.ts [hunk 2]
```diff
@@ -560,11 +560,20 @@ export class JanusVideoRoomService {
...
+        } else if(response.name === 'PermissionDeniedError' || response.name === 'PermissionDismissedError') {
+          var msg = `Error: ${response.message} <br />
+            Please allow media permissions.
+            <a href="https://support.google.com/chrome/answer/2693767?hl=en">
+              For more details.
+            </a>`;          
+          this.toastr.error(msg);
+        } else if(response.name === 'MediaDeviceNotSupported') {          
+          this.$log.error('Error creating SDP offer', response);
+          this.toastr.error(`Bummers, can't share video: ${response.message}`);
```
Comment (kkavod) [inline frontend/src/app/components/janus/janusVideoRoom.service.ts]: Why it reports and shows same error messages as the else below?

File: frontend/src/app/components/auth/login.controller.ts [hunk 3]
```diff
@@ -20,8 +20,8 @@ export class LoginController {
     }).then((user: IUser) => {
       this.$mdDialog.hide(user);
     }).catch((response: any) => {
-      this.$log.error('Error submitting login form', response);
-      this.toastr.error(response.errors.join(' '));
+      /*this.$log.error('Error submitting login form', response);
+      this.toastr.error(response.errors.join(' '));*/
```
Comment (kkavod) [inline frontend/src/app/components/auth/login.controller.ts]: Why commented out?

File: frontend/src/app/components/auth/auth.service.ts [hunk 4]
```diff
@@ -71,7 +71,6 @@ export class AuthService {
     return this.$auth.signOut()
       .then(this.onLogout)
       .catch((resp: any) => {
-        this.$log.error('Error signing out', resp);
```
Comment (kkavod) [inline frontend/src/app/components/auth/auth.service.ts]: Why removed?

File: frontend/src/app/components/janus/janusVideoRoom.service.ts (hunk 2 above)
Comment (davgur) [inline frontend/src/app/components/janus/janusVideoRoom.service.ts]: else if(response.name === 'MediaDeviceNotSupported') {          
          this.$log.error(response.name, response);
          this.toastr.error(`Your browser not support video. </br>
            Please use 
            <a href="//www.google.com/chrome/browser/desktop/">
              chrome
            </a>.`);
        }
?

File: frontend/src/app/components/janus/janus.service.ts (hunk 1 above)
Comment (davgur) [inline frontend/src/app/components/janus/janus.service.ts]: OK

File: frontend/src/app/components/janus/janusVideoRoom.service.ts (hunk 2 above)
Comment (davgur) [inline frontend/src/app/components/janus/janusVideoRoom.service.ts]: I want add (in the 573 line). That mean - don't do nothing if was resolved before.
what you think?
else if(response !== undefined) {
          this.$log.error('Error creating SDP offer', response);
          this.toastr.error(`Bummers, can't share video: ${response.message}`);
        }

File: frontend/src/app/components/auth/auth.service.ts (hunk 4 above)
Comment (davgur) [inline frontend/src/app/components/auth/auth.service.ts]: you right. I return it back.

File: frontend/src/app/components/auth/login.controller.ts (hunk 3 above)
Comment (davgur) [inline frontend/src/app/components/auth/login.controller.ts]: .catch((response: any) => {
      var msg = response.errors.join(' ');
      if(msg !== "Invalid credentials"){
        this.$log.error('Error submitting login form', response);
      }
      this.toastr.error(msg);
    });

**Pull request merged**